lib/modules/manager/cargo/extract.ts

```diff
@@ -184,11 +189,27 @@ export async function extractPackageFile(
       targetDeps = targetDeps.concat(deps);
     });
   }
+
+  const workspaceSection = cargoManifest.workspace;
+  let workspaceDeps: PackageDependency[] = [];
+  if (workspaceSection) {
+    const target = 'dependencies';
+    // workspace depedencies could potentially also be fully qualified
+    // like target.*.dependencies. It would require further unfolding of the sections
+    workspaceDeps = extractFromSection(
+      workspaceSection,
+      target,
+      cargoRegistries,
+      target
```

Comment:
```suggestion
      cargoRegistries
```

Reply:
Done. This required a change in the tests as well, as the output target isn't marked as a depedency. Just fyi